### PR TITLE
[Maintenance]: Sync Practice Exercises from Problem Specification Repo

### PR DESCRIPTION
**Pull request by BethanyG**
**Reviewers Please Note:**  Instruction files have bee pulled from [problem specifications](https://github.com/exercism/problem-specifications).  Any spelling, grammar, or other errors need to be corrected there.

Decided to keep this as one PR.  Breaking it down by exercise or exercise group seemed more trouble/noise.

Synced the following exercise instruction files:

- [x] affine-cipher
- [x] all-your-base
- [x] allergies
- [x] anagram
- [x] atbash-cipher
- [x] book-store
- [x] crypto-square
- [x] diffie-hellman
- [x] dominoes
- [x] go-counting
-  [x] grade-school
- [x] grains
- [x] grep
- [x] largest-series-product
- [x] meetup
- [x] minesweeper
- [x] nucleotide-count
- [x] ocr-numbers
- [x] phone-number
- [x] point-mutations
- [x] pov
- [x] prime-factors
- [x] pythagorean-triplet
- [x] queen-attack
- [x] raindrops
- [x] rational-numbers
- [x] resistor-color
- [x] rest-api
- [x] roman-numerals
- [x] saddle-points
- [x] satellite
- [x] scale-generator
- [x] sgf-parsing
- [x] sieve
- [x] space-age
- [x] spiral-matrix
- [x] triangle
- [x] two-bucket
- [x] yacht
____

Synced the following exercise `config.json` files:

- [x] acronym
- [x] armstrong-numbers
- [x] binary
- [x] bowling
- [x] change
- [x] collatz-conjecture
- [x] connect
- [x] darts
- [x] dnd-character
- [x] dot-dsl
- [x] error-handling
- [x] flatten-array
- [x] food-chain
- [x] forth
- [x] grade-school
- [x] hello-world
- [x] high-scores
- [x] linked-list
- [x] list-ops
- [x] markdown
- [x] minesweeper
- [x] paasio
- [x] pig-latin
- [x] pov
- [x] resistor-color
- [x] reverse-string
- [x] scale-generator
- [x] simple-cipher
- [x] simple-linked-list
- [x] twelve-days
- [x] two-fer
- [x] yacht
____

Synced the following exercise `tests.toml` files, regenerated the `*_tests.py` files, tested the `example.py` files, and edited solutions as needed: 

- [x] accumulate (_deprecated_)
- [x] bowling
- [x] collatz conjecture
- [x] flatten array
- [x] fourth
- [x] largest series product
- [x] list ops
- [x] matching brackets
- [x] meetup
- [x] pangram
- [x] protein translation
- [x] rational numbers (_also edited example.py_)
- [x] roman numerals
- [x] scale generator (_also edited example.py_)
- [x] space age
- [x] two bucket
- [x] word search
- [x] zipper

### CODE REVIEW

File: exercises/practice/diffie-hellman/.docs/instructions.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ the same to pick a private key b.
 
 Alice calculates a public key A.
 
-    A = g**a mod p
+    A = gᵃ mod p
```
Comment (neenjaw) [inline exercises/practice/diffie-hellman/.docs/instructions.md:22]: Interesting, i didn't realize that Unicode or markdown would handle this.

File: exercises/practice/pangram/pangram_test.py [hunk 2]
```diff
@@ -40,8 +40,8 @@ def test_missing_letters_replaced_by_numbers(self):
     def test_mixed_case_and_punctuation(self):
         self.assertIs(is_pangram('"Five quacking Zephyrs jolt my wax bed."'), True)
 
-    def test_case_insensitive(self):
-        self.assertIs(is_pangram("the quick brown fox jumps over with lazy FX"), False)
+    def test_a_m_and_a_m_are_26_different_characters_but_not_a_pangram(self):
```
Comment (neenjaw) [inline exercises/practice/pangram/pangram_test.py:43]: 👀 while maybe not true snake case, using `A_M` for the second range may may more sense in context?

File: exercises/practice/two-bucket/two_bucket.py [hunk 3]
```diff
@@ -1,2 +1,55 @@
+# def measure(bucket_one, bucket_two, goal, start_bucket):
```
Comment (neenjaw) [inline exercises/practice/two-bucket/two_bucket.py:1]: Shouldn't this file be stubs and example.py be this solution?

**Review by neenjaw — APPROVED** [review]
Approving as I think the majority of the work is top notch. Impressive that with all those updates you only had to update 2 example solutions. 

1 noted concern to double check before merging: the stub for two-bucket seems to have been replaced with the example solution. Is this intentional?

Nice work!

File: exercises/practice/diffie-hellman/.docs/instructions.md (hunk 1 above)
Comment (BethanyG) [reply exercises/practice/diffie-hellman/.docs/instructions.md:22]: There was **quite** the discussion on that one!  😄   [diffie-hellman](https://github.com/exercism/problem-specifications/pull/1933).

File: exercises/practice/pangram/pangram_test.py (hunk 2 above)
Comment (BethanyG) [reply exercises/practice/pangram/pangram_test.py:43]: 🤔   Yeah.  Problem is that this is auto-generated, and I don't know if I can get the generator to fix it.  Let me try under a separate PR, if that's OK?

File: exercises/practice/two-bucket/two_bucket.py (hunk 3 above)
Comment (BethanyG) [reply exercises/practice/two-bucket/two_bucket.py]: GAH!  **good catch!**.  Pushed fix.  Thank you so much for being my safety net!!

**Pull request merged**